thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/utils/AnomalyUtils.java

```diff
@@ -48,4 +50,21 @@ public static void logAnomaliesOverlapWithWindow(DateTime windowStart, DateTime
           windowStart, windowEnd, sb.toString());
     }
   }
+
+  /**
+   * This function checks if the input list of merged anomalies has at least one positive label.
+   * It is a helper for alert filter auto tuning
+   * @param mergedAnomalyResultDTOS
+   * @return true if the list of merged anomalies has at least one positive label, false otherwise
+   */
+  public static Boolean checkHasPostiveLabels(List<MergedAnomalyResultDTO> mergedAnomalyResultDTOS){
+    for(MergedAnomalyResultDTO anomaly: mergedAnomalyResultDTOS){
+      AnomalyFeedback feedback = anomaly.getFeedback();
+      boolean label = !(feedback == null || feedback.getFeedbackType() == AnomalyFeedbackType.NOT_ANOMALY);
```

Comment:
Minor suggestion: Change to 
    boolean label = (feedback != null && (feedback.getFeedbackType() == AnomalyFeedbackType.ANOMALY || feedback.getFeedbackType() == AnomalyFeedbackType.ANOMALY_NO_ACTION);

Because we might add other types of anomalies in the future.